DOCKER.md

```diff
@@ -125,17 +125,23 @@ you can provide your own
 [config.json](https://github.com/scality/S3/blob/master/config.json)
 using [Docker Volume](https://docs.docker.com/engine/tutorials/dockervolumes/).
 
-First add a new key-value pair in the regions section of your config.json.
-The key in the key-value pair should be your "region" name and the value
-is an array containing any host name you would like to add:
+First add a new key-value pair in the restEndpoints section of your config.json.
+The key in the key-value pair should be the host name you would like to add
+and the value is the location_constrain used for this endpoint.
```

Comment:
also, I would say the value is the "default" location_constraint for this endpoint